include/deal.II/fe/fe_values.h

```diff
@@ -1663,9 +1663,9 @@ class FEValuesBase : protected FEValuesData<dim,spacedim>,
    *
    * @dealiiRequiresUpdateFlags{update_values}
    */
-  template <class InputVector, typename number>
+  template <class InputVector>
   void get_function_values (const InputVector &fe_function,
-                            std::vector<number> &values) const;
+                            std::vector<typename InputVector::value_type> &values) const;
```

Comment:
apparently, one of `get_function_values` was already `number`-templated. I do not know why was it the case, but supposedly to evaluate, say, `double` global vector in `float` values. Shall I keep these changes?